bigquery/storage/managedwriter/flow_controller.go

```diff
@@ -50,6 +50,17 @@ func newFlowController(maxInserts, maxInsertBytes int) *flowController {
 	return fc
 }
 
+// copyFlowController is for creating a new flow controller based on
+// settings from another.  It does not copy flow state.
+func copyFlowController(in *flowController) *flowController {
```

Comment:
should this function go to the test file as is only being used there or there are plans to use it later ?

Reply:
This is in prep of future refactors, it will be used for replicating the flow controller as we spin up new connections.